### DISCUSSION TITLE
Updating CI to run on `ubuntu-22.04`

### DISCUSSION THREAD

**Discussion by realeinherjar** [rust-bitcoin/rust-bitcoin#2114] [General]
Are we running on `ubuntu-20.04` by any specific reason?
Could we update to `ubuntu-22.04` or even `ubuntu-latest` (latest LTS)?

I think that CI should be ran on Ubuntu latest LTS (which is `22.04`)

What do you guys think?

If we have a green light I can do a PR...

**Comment by vincenzopalazzo** [comment]
why this is needed? rustc runs on `ubuntu-20.04` as well

**Comment by apoelstra** [comment]
Sure. There is no specific reason, we just don't ever update our CI scripts unless somebody prods us into it.

Though I am also curious if there's a specific reason we *should* update.

**Comment by realeinherjar** [comment]
Another compelling argument is that only the `fuzz.yml` runs on `ubuntu-20.04` all other CI workflows are `ubuntu-latest`

**Comment by vincenzopalazzo** [comment]
>Another compelling argument is that only the fuzz.yml runs on ubuntu-20.04 all other CI workflows are ubuntu-latest

I think it is not a problem where we run the CI because we run only cargo and it works everywhere, but you are already changing some code around the CI file so for me it is also fine to run on the latest

**Reply by realeinherjar** [reply]
I'll incorporate this to #2116

**Comment by realeinherjar** [comment]
Closing this in favor of #2116
